package.json

```diff
@@ -52,6 +52,7 @@
     "@babel/plugin-proposal-export-default-from": "^7.17.12",
     "@babel/preset-env": "7.8.7",
     "@cocos/babel-preset-cc": "2.2.0",
+    "@cocos/build-engine": "4.4.2",
```

Comment:
please remove this dependencies, for now `@cocos/build-engine` is a local package, not the pacakge on npm

after remove this, please also update the package-lock.json

Reply:
fixed